Formula/a/asymptote.rb

```diff
@@ -39,10 +39,13 @@ class Asymptote < Formula
   end
 
   resource "manual" do
-    url "https://downloads.sourceforge.net/project/asymptote/2.92/asymptote.pdf"
-    sha256 "ba96b049da98207eed38eb3be5c306fbd10ff8b00124e26609d5effe7adbdac6"
+    url "https://downloads.sourceforge.net/project/asymptote/2.95/asymptote.pdf"
+    sha256 "6fa4428a78c6af413ed82173056dd6330a496c5f7e930883b16c5cbfc01394cf"
   end
 
+  # Remove un-wanted non-executables
+  patch :DATA
```

Comment:
maybe log an upstream issue for this?

Reply:
I think this is a homebrew specific problem and so I hesitate to open an issue. Maybe I can ask to upstream to add an option or any other ways to achieve the patch.